server/data/incidentReportingApi.test.ts

```diff
@@ -1,7 +1,7 @@
 import nock from 'nock'
+import { SanitisedError } from '@ministryofjustice/hmpps-rest-client'
 
 import config from '../config'
-import type { SanitisedError } from '../sanitisedError'
```

Comment:
Bear with me as I thought I also got rid of the `SanitisedError` interface but I didn't which is a bit confusing as it's slightly different from the type in the library (library one has `responseStatus` not `status`).